Comparing source compatibility of C:\au\builds\uimaj-3.0.0\rc4\artifacts\target\checkout\uimaj-core\target\uima-core.jar against C:\Users\MarshallSchor\.m2\repository\org\apache\uima\uimaj-core\2.10.2\uimaj-core-2.10.2.jar
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.uima.analysis_component.AnalysisComponent_ImplBase  (not serializable)
	+++  NEW METHOD: PROTECTED(+) org.apache.uima.util.Logger getLogger()
***  MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.analysis_engine.AnalysisEngine  (not serializable)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String PARAM_THROTTLE_EXCESSIVE_ANNOTATOR_LOGGING
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MDC_ANNOTATOR_CONTEXT_NAME
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MDC_ANNOTATOR_IMPL_NAME
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MDC_ROOT_CONTEXT_ID
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MDC_CAS_ID
===* UNCHANGED CLASS: PUBLIC org.apache.uima.analysis_engine.AnalysisEngineProcessException  (compatible)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.analysis_engine.annotator.AnnotatorConfigurationException  (compatible)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.analysis_engine.annotator.AnnotatorContextException  (compatible)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.analysis_engine.annotator.AnnotatorInitializationException  (compatible)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.analysis_engine.annotator.AnnotatorProcessException  (compatible)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.analysis_engine.ResultNotSupportedException  (compatible)
***! MODIFIED CLASS: PUBLIC org.apache.uima.cas.admin.CASAdminException  (superclass modified)
	+++  NEW INTERFACE: org.apache.uima.I18nExceptionI
	***  MODIFIED SUPERCLASS: org.apache.uima.UIMARuntimeException (<- java.lang.RuntimeException)
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) TYPE_SYSTEM_LOCKED
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) BAD_FEATURE_SYNTAX
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) MUST_COMMIT_TYPE_SYSTEM
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) DESERIALIZATION_ERROR
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) DUPLICATE_FEATURE
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) TYPE_IS_FEATURE_FINAL
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) MUST_COMMIT_INDEX_REPOSITORY
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) JCAS_ERROR
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) BAD_TYPE_SYNTAX
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) FLUSH_DISABLED
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) TYPE_IS_INH_FINAL
	***! MODIFIED FIELD: PUBLIC STATIC FINAL java.lang.String (<- int) REPOSITORY_LOCKED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String DUPLICATE_TYPE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String STRING_SUBTYPE_REDEFINE_NAME_CONFLICT
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String TOO_MANY_TYPES
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String STRING_SUBTYPE_CONFLICTING_ALLOWED_VALUES
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String INDEX_DUPLICATES_NOT_SUPPORTED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MISSING_ARRAY_TYPE_FOR_COMPONENT
	---! REMOVED CONSTRUCTOR: PUBLIC(-) CASAdminException(int)
	+++  NEW CONSTRUCTOR: PUBLIC(+) CASAdminException(java.lang.String, java.lang.Throwable, java.lang.String, java.lang.Object[])
	+++  NEW CONSTRUCTOR: PUBLIC(+) CASAdminException(java.lang.String, java.lang.Object[])
	+++  NEW CONSTRUCTOR: PUBLIC(+) CASAdminException(java.lang.Throwable, java.lang.String, java.lang.Object[])
	---! REMOVED METHOD: PUBLIC(-) boolean addArgument(java.lang.String)
	---! REMOVED METHOD: PUBLIC(-) java.lang.String[] getArguments()
	---! REMOVED METHOD: PUBLIC(-) java.lang.String getBundleShortName()
	---! REMOVED METHOD: PUBLIC(-) int getError()
	---  REMOVED METHOD: PUBLIC(-) java.lang.String getMessage()
	---! REMOVED METHOD: PUBLIC(-) java.lang.String getMessageCode()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getResourceBundleName()
	---  REMOVED METHOD: PUBLIC(-) java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC ABSTRACT org.apache.uima.cas.admin.CASFactory  (not serializable)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.uima.cas.admin.CASMgr createCAS(int, boolean)
		+++  NEW ANNOTATION: java.lang.Deprecated
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.admin.CASMgr  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setCAS(org.apache.uima.cas.CAS)
		+++  NEW ANNOTATION: java.lang.Deprecated
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.admin.LinearTypeOrder  (not serializable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) int compare(org.apache.uima.cas.FeatureStructure, org.apache.uima.cas.FeatureStructure)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) boolean isEmptyTypeOrder()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.admin.TypeSystemMgr  (not serializable)
	***! MODIFIED METHOD: PUBLIC ABSTRACT org.apache.uima.cas.TypeSystem (<-void) commit()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.TypeSystem commit(java.lang.ClassLoader)
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.ArrayFS  (not serializable)
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	***! MODIFIED METHOD: PUBLIC ABSTRACT org.apache.uima.jcas.cas.TOP (<-org.apache.uima.cas.FeatureStructure) get(int)
		===  UNCHANGED EXCEPTION: java.lang.ArrayIndexOutOfBoundsException
	---  REMOVED METHOD: PUBLIC(-) ABSTRACT(-) int size()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.jcas.cas.TOP[] toArray(org.apache.uima.jcas.cas.TOP[])
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.CAS  (not serializable)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String UIMA_TCAS_PREFIX
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String FEATURE_BASE_NAME_INT_ARRAY
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String FEATURE_BASE_NAME_FS_ARRAY
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.CommonArrayFS emptyArray(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.BooleanArray emptyBooleanArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.ByteArray emptyByteArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.DoubleArray emptyDoubleArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.FloatArray emptyFloatArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyFloatList emptyFloatList()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.FSArray emptyFSArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyFSList emptyFSList()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.IntegerArray emptyIntegerArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyIntegerList emptyIntegerList()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyList emptyList(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.LongArray emptyLongArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.ShortArray emptyShortArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.StringArray emptyStringArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyStringList emptyStringList()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FSIterator getAllIndexedFS(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FSIterator getAllIndexedFS(org.apache.uima.cas.Type)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.text.AnnotationIndex getAnnotationIndex(java.lang.Class)
		+++  NEW EXCEPTION: org.apache.uima.cas.CASRuntimeException
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.Type getCasType(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) java.util.Collection getIndexedFSs(org.apache.uima.cas.Type)
	+++* NEW METHOD: PUBLIC(+) java.util.Collection getIndexedFSs()
	+++* NEW METHOD: PUBLIC(+) java.util.Collection getIndexedFSs(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.impl.JCasImpl getJCasImpl()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(org.apache.uima.cas.Type)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(int)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(java.lang.String)
**** MODIFIED CLASS: PUBLIC org.apache.uima.cas.CASException  (serialVersionUID modified)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_GETTER_SETTER_MISSING
***! MODIFIED CLASS: PUBLIC org.apache.uima.cas.CASRuntimeException  (serialVersionUID modified)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_FIELD_MISSING_IN_TYPE_SYSTEM
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String DESERIALIZING_V2_DELTA_V3
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_MISSING_FIELD_ACCESSOR
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_CAS_MISMATCH_SUPERTYPE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String DESER_FORM_6_BAD_TYPE_CODE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String ANNOTATION_INDEX_REQUIRED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String CREATE_FS_BEFORE_TS_COMMITTED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String FS_NOT_MEMBER_OF_CAS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String ADD_ARRAY_TYPE_AFTER_TS_COMMITTED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String TYPESYSTEMS_NOT_COMMITTED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SUBITERATOR_AMBIGUOUS_POSITION_DIFFERENT_TYPES
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String INVALID_FS_ID
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String deserialized_type_not_found
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_TYPE_RANGE_MISMATCH
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SELECT_ALT_SRC_INVALID
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String CAS_MISSING_FS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SELECT_GET_TOO_MANY_INSTANCES
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String ILLEGAL_ADD_TO_INDEX_IN_BASE_CAS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SELECT_GET_NO_INSTANCES
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_MISSING_TYPEINDEX
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String ILLEGAL_TYPE_CHANGE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_MISMATCH_SUPERTYPE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_MISSING_GETTER
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_CAS_NOT_V3
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_FIELD_ADJ_OFFSET_CHANGED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String INAPPROP_FEAT_X
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_ARRAY_NOT_SUPPORTED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_ALPHA_LEVEL_NOT_SUPPORTED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String ILLEGAL_TYPE_CHANGE_IN_INDEX
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String GET_CLASS_FOR_TYPE_BEFORE_TS_COMMIT
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String ILLEGAL_SOFAREF_MODIFICATION
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SET_REF_FROM_STRING_NOT_SUPPORTED
	---! REMOVED CONSTRUCTOR: PUBLIC(-) CASRuntimeException(java.lang.String)
	+++  NEW CONSTRUCTOR: PUBLIC(+) CASRuntimeException(java.lang.Throwable, java.lang.String, java.lang.Object[])
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.CommonArrayFS  (not serializable)
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++* NEW METHOD: PUBLIC(+) java.lang.String getValuesAsCommaSeparatedString()
	+++* NEW METHOD: PUBLIC(+) boolean isEmpty()
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) java.lang.String[] toStringArray()
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.FeaturePath  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.uima.cas.TypeClass getTypClass(org.apache.uima.cas.FeatureStructure)
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.TypeClass getTypeClass(org.apache.uima.cas.FeatureStructure)
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.FeatureStructure  (not serializable)
	+++  NEW INTERFACE: java.lang.Cloneable
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) int _getTypeCode()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) int _id()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) int getAddress()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.JCas getJCas()
+++  NEW ENUM: PUBLIC(+) FINAL(+) org.apache.uima.cas.FSComparators  (compatible)
	+++  NEW INTERFACE: java.lang.Comparable
	+++  NEW INTERFACE: java.io.Serializable
	+++  NEW SUPERCLASS: java.lang.Enum
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.cas.FSComparators WITHOUT_ID
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.cas.FSComparators WITH_ID
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.cas.FSComparators WITH_TYPE_ORDER
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.cas.FSComparators WITHOUT_TYPE_ORDER
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.cas.FSComparators valueOf(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.cas.FSComparators[] values()
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.FSIndex  (not serializable)
	+++  NEW INTERFACE: java.util.Collection
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) org.apache.uima.cas.FSIterator iterator(org.apache.uima.cas.FeatureStructure)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs select()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs select(org.apache.uima.cas.Type)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs select(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs select(int)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs select(java.lang.String)
	+++* NEW METHOD: PUBLIC(+) java.util.stream.Stream stream()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FSIndex subType(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FSIndex subType(org.apache.uima.cas.Type)
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.FSIndexRepository  (not serializable)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FSIterator getAllIndexedFS(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) java.util.Collection getIndexedFSs(org.apache.uima.cas.Type)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) java.util.Collection getIndexedFSs(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) java.util.Collection getIndexedFSs()
	+++* NEW METHOD: PUBLIC(+) void removeAllExcludingSubtypes(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) void removeAllIncludingSubtypes(java.lang.Class)
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.FSIterator  (not serializable)
	+++  NEW INTERFACE: java.util.ListIterator
	+++* NEW METHOD: PUBLIC(+) void add(org.apache.uima.cas.FeatureStructure)
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) org.apache.uima.cas.FeatureStructure get()
		===  UNCHANGED EXCEPTION: java.util.NoSuchElementException
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure getNvc()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.Type getType()
	+++* NEW METHOD: PUBLIC(+) boolean hasNext()
	+++* NEW METHOD: PUBLIC(+) boolean hasPrevious()
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) void moveToNext()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void moveToNextNvc()
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) void moveToPrevious()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void moveToPreviousNvc()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FeatureStructure next()
	+++* NEW METHOD: PUBLIC(+) int nextIndex()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FeatureStructure nextNvc()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FeatureStructure previous()
	+++* NEW METHOD: PUBLIC(+) int previousIndex()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FeatureStructure previousNvc()
	+++* NEW METHOD: PUBLIC(+) void remove()
	+++* NEW METHOD: PUBLIC(+) void set(org.apache.uima.cas.FeatureStructure)
	+++* NEW METHOD: PUBLIC(+) java.util.Spliterator spliterator()
	+++* NEW METHOD: PUBLIC(+) java.util.stream.Stream stream()
***  MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.IntArrayFS  (not serializable)
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	***  MODIFIED METHOD: PUBLIC ABSTRACT void set(int, int)
		---  REMOVED EXCEPTION: java.lang.ArrayIndexOutOfBoundsException
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs  (not serializable)
	+++  NEW INTERFACE: java.util.stream.Stream
	+++  NEW INTERFACE: java.util.stream.BaseStream
	+++  NEW INTERFACE: java.lang.AutoCloseable
	+++  NEW INTERFACE: java.lang.Iterable
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs allViews()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs allViews(boolean)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure[] asArray(java.lang.Class)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.util.List asList()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs at(org.apache.uima.cas.text.AnnotationFS)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs at(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs backwards()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs backwards(boolean)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs between(org.apache.uima.cas.text.AnnotationFS, org.apache.uima.cas.text.AnnotationFS)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs coveredBy(org.apache.uima.cas.text.AnnotationFS)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs coveredBy(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs covering(org.apache.uima.cas.text.AnnotationFS)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs covering(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs following(org.apache.uima.jcas.tcas.Annotation)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs following(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs following(org.apache.uima.jcas.tcas.Annotation, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs following(int, int)
	+++  NEW METHOD: PUBLIC(+) void forEach(java.util.function.Consumer)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FSIterator fsIterator()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure get()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure get(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure get(org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure get(org.apache.uima.jcas.cas.TOP, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure get(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure get(int, int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs includeAnnotationsWithEndBeyondBounds()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs includeAnnotationsWithEndBeyondBounds(boolean)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs limit(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs nonOverlapping()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs nonOverlapping(boolean)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs nullOK()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs nullOK(boolean)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs orderNotNeeded()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs orderNotNeeded(boolean)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs preceding(org.apache.uima.jcas.tcas.Annotation)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs preceding(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs preceding(org.apache.uima.jcas.tcas.Annotation, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs preceding(int, int)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.cas.SelectFSs select(org.apache.uima.cas.FSIndex)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs shifted(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure single()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure single(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure single(org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure single(org.apache.uima.jcas.cas.TOP, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure single(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure single(int, int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure singleOrNull()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure singleOrNull(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure singleOrNull(org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure singleOrNull(org.apache.uima.jcas.cas.TOP, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure singleOrNull(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure singleOrNull(int, int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs skipWhenSameBeginEndType()
	+++  NEW METHOD: PUBLIC(+) java.util.Spliterator spliterator()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs startAt(org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs startAt(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs startAt(org.apache.uima.jcas.cas.TOP, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs startAt(int, int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs typePriority()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs typePriority(boolean)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.SelectFSs useAnnotationEquals(boolean)
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.SofaFS  (not serializable)
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) int getSofaNum()
	***  MODIFIED METHOD: PUBLIC ABSTRACT void setRemoteSofaURI(java.lang.String)
		---  REMOVED EXCEPTION: org.apache.uima.cas.CASRuntimeException
***  MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.StringArrayFS  (not serializable)
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	---  REMOVED METHOD: PUBLIC(-) ABSTRACT(-) void copyFromArray(java.lang.String[], int, int, int)
		---  REMOVED EXCEPTION: java.lang.ArrayIndexOutOfBoundsException
	---  REMOVED METHOD: PUBLIC(-) ABSTRACT(-) void copyToArray(int, java.lang.String[], int, int)
		---  REMOVED EXCEPTION: java.lang.ArrayIndexOutOfBoundsException
	---  REMOVED METHOD: PUBLIC(-) ABSTRACT(-) int size()
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.text.AnnotationFS  (not serializable)
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void setBegin(int)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void setEnd(int)
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.Type  (not serializable)
	+++  NEW INTERFACE: java.lang.Iterable
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) boolean isStringOrStringSubtype()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) boolean isStringSubtype()
	+++* NEW METHOD: PUBLIC(+) boolean subsumes(org.apache.uima.cas.Type)
***  MODIFIED ENUM: PUBLIC FINAL org.apache.uima.cas.TypeClass  (compatible)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.cas.TypeClass TYPE_CLASS_JAVAOBJECT
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.cas.TypeClass TYPE_CLASS_JAVAOBJECTARRAY
***  MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.TypeSystem  (not serializable)
	+++  NEW INTERFACE: java.lang.Iterable
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String PARAM_EXCLUDE_PREBUILT_TYPES
===* UNCHANGED CLASS: PUBLIC org.apache.uima.collection.CollectionException  (compatible)
***! MODIFIED CLASS: PUBLIC org.apache.uima.examples.SourceDocumentInformation  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.AnnotationImpl
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.text.AnnotationFS
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.AnnotationBaseImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_documentSize
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_offsetInSource
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_lastSegment
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_uri
	---! REMOVED CONSTRUCTOR: PUBLIC(-) SourceDocumentInformation(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) SourceDocumentInformation(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.uima.flow.FlowController_ImplBase  (not serializable)
	+++  NEW METHOD: PROTECTED(+) org.apache.uima.util.Logger getLogger()
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.I18nExceptionI  (not serializable)
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.Object[] getArguments()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.Throwable getCause()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String getLocalizedMessage()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getLocalizedMessage(java.util.Locale)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String getMessage()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String getMessageKey()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String getResourceBundleName()
	+++  NEW METHOD: PUBLIC(+) boolean hasMessageKey(java.lang.String)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.AnnotationBase  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.cas.impl.AnnotationBaseImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_sofa
	---! REMOVED CONSTRUCTOR: PUBLIC(-) AnnotationBase(int, org.apache.uima.jcas.cas.TOP_Type)
	===  UNCHANGED CONSTRUCTOR: PROTECTED AnnotationBase()
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW CONSTRUCTOR: PUBLIC(+) AnnotationBase(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	***! MODIFIED METHOD: PUBLIC org.apache.uima.jcas.cas.Sofa (<-org.apache.uima.cas.SofaFS) getSofa()
	+++  NEW METHOD: PUBLIC(+) void setFeatureValue(org.apache.uima.cas.Feature, org.apache.uima.cas.FeatureStructure)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.jcas.cas.ArrayFSImpl  (not serializable)
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: org.apache.uima.cas.CommonArrayFS
	+++  NEW INTERFACE: org.apache.uima.cas.ArrayFS
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW ANNOTATION: java.lang.Deprecated
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.BooleanArray  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonPrimitiveArray
	+++  NEW INTERFACE: org.apache.uima.cas.impl.BooleanArrayFSImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) BooleanArray(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) BooleanArray(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int)
	+++  NEW METHOD: PUBLIC(+) boolean[] _getTheArray()
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.BooleanArray create(org.apache.uima.jcas.JCas, boolean[])
	+++  NEW METHOD: PUBLIC(+) void setArrayValueFromString(int, java.lang.String)
	---  REMOVED METHOD: PUBLIC(-) java.lang.String[] toStringArray()
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.ByteArray  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.cas.impl.ByteArrayFSImpl
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonPrimitiveArray
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) ByteArray(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) ByteArray(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int)
	+++  NEW METHOD: PUBLIC(+) byte[] _getTheArray()
	+++  NEW METHOD: PUBLIC(+) boolean contains(byte)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.ByteArray create(org.apache.uima.jcas.JCas, byte[])
	+++  NEW METHOD: PUBLIC(+) void setArrayValueFromString(int, java.lang.String)
	---  REMOVED METHOD: PUBLIC(-) java.lang.String[] toStringArray()
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.jcas.cas.CommonList  (not serializable)
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_tail
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.util.List EMPTY_LIST_STRING
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_head
	+++  NEW METHOD: PUBLIC(+) void anyListToOutput(org.apache.uima.cas.impl.XmiSerializationSharedData, org.apache.uima.cas.impl.CasSerializerSupport$CasDocSerializer, java.util.function.Consumer)
	+++  NEW METHOD: PUBLIC(+) java.util.List anyListToStringList(org.apache.uima.cas.impl.XmiSerializationSharedData, org.apache.uima.cas.impl.CasSerializerSupport$CasDocSerializer)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.jcas.cas.CommonList createNonEmptyNode()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.jcas.cas.CommonList emptyList()
	+++  NEW METHOD: PUBLIC(+) java.lang.String get_headAsString()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.CommonList getCommonTail()
	+++  NEW METHOD: PUBLIC(+) int getLength()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.CommonList getNonEmptyNthNode(int)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.CommonList getNthNode(int)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.CommonList insertNode()
	+++  NEW METHOD: PUBLIC(+) boolean isEmpty()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.CommonList pushNode()
	+++  NEW METHOD: PUBLIC(+) void set_headFromString(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void setTail(org.apache.uima.jcas.cas.CommonList)
	+++  NEW METHOD: PUBLIC(+) void walkList(java.util.function.Consumer, java.lang.Runnable)
	+++  NEW METHOD: PUBLIC(+) void walkList_saxException(org.apache.uima.internal.util.function.Consumer_withSaxException, java.lang.Runnable)
		+++  NEW EXCEPTION: org.xml.sax.SAXException
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.jcas.cas.CommonPrimitiveArray  (not serializable)
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: org.apache.uima.cas.CommonArrayFS
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void setArrayValueFromString(int, java.lang.String)
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.DoubleArray  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonPrimitiveArray
	+++  NEW INTERFACE: org.apache.uima.cas.impl.DoubleArrayFSImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) DoubleArray(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) DoubleArray(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int)
	+++  NEW METHOD: PUBLIC(+) double[] _getTheArray()
	+++  NEW METHOD: PUBLIC(+) boolean contains(double)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.DoubleArray create(org.apache.uima.jcas.JCas, double[])
	+++  NEW METHOD: PUBLIC(+) void forEach(java.util.function.DoubleConsumer)
	+++  NEW METHOD: PUBLIC(+) void setArrayValueFromString(int, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) java.util.Spliterator$OfDouble spliterator()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.DoubleStream stream()
	---  REMOVED METHOD: PUBLIC(-) java.lang.String[] toStringArray()
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.EmptyFloatList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.EmptyList
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) EmptyFloatList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) EmptyFloatList(org.apache.uima.cas.impl.TypeImpl_list)
	+++  NEW CONSTRUCTOR: PUBLIC(+) EmptyFloatList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.EmptyFSList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.EmptyList
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) EmptyFSList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) EmptyFSList(org.apache.uima.cas.impl.TypeImpl_list)
	+++  NEW CONSTRUCTOR: PUBLIC(+) EmptyFSList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.EmptyIntegerList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.EmptyList
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) EmptyIntegerList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) EmptyIntegerList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW CONSTRUCTOR: PUBLIC(+) EmptyIntegerList(org.apache.uima.cas.impl.TypeImpl_list)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.jcas.cas.EmptyList  (not serializable)
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW SUPERCLASS: java.lang.Object
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.EmptyStringList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.EmptyList
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) EmptyStringList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) EmptyStringList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW CONSTRUCTOR: PUBLIC(+) EmptyStringList(org.apache.uima.cas.impl.TypeImpl_list)
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.FloatArray  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonPrimitiveArray
	+++  NEW INTERFACE: org.apache.uima.cas.impl.FloatArrayFSImpl
	+++  NEW INTERFACE: java.lang.Iterable
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) FloatArray(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FloatArray(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int)
	+++  NEW METHOD: PUBLIC(+) float[] _getTheArray()
	+++  NEW METHOD: PUBLIC(+) boolean contains(float)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.FloatArray create(org.apache.uima.jcas.JCas, float[])
	+++  NEW METHOD: PUBLIC(+) java.util.Iterator iterator()
	+++  NEW METHOD: PUBLIC(+) void setArrayValueFromString(int, java.lang.String)
	---  REMOVED METHOD: PUBLIC(-) java.lang.String[] toStringArray()
***! MODIFIED CLASS: PUBLIC ABSTRACT (<- NON_ABSTRACT) org.apache.uima.jcas.cas.FloatList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonList
	---  REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) int typeIndexID
	---  REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) int type
	---! REMOVED CONSTRUCTOR: PUBLIC(-) FloatList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FloatList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) boolean contains(float)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.FloatList create(org.apache.uima.jcas.JCas, java.lang.Float[])
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.NonEmptyFloatList createNonEmptyNode()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyFloatList emptyList()
	---  REMOVED METHOD: PUBLIC(-) int getTypeIndexID()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.Stream stream()
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.FSArray  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===! UNCHANGED INTERFACE: org.apache.uima.cas.ArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.SelectViaCopyToArray
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.ArrayFSImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) FSArray(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSArray(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP[] _getTheArray()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.FeatureStructure[] _toArrayForSelect()
	+++  NEW METHOD: PUBLIC(+) boolean contains(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.FSArray create(org.apache.uima.jcas.JCas, org.apache.uima.cas.FeatureStructure[])
	***! MODIFIED METHOD: PUBLIC org.apache.uima.jcas.cas.TOP (<-org.apache.uima.cas.FeatureStructure) get(int)
	+++  NEW METHOD: PUBLIC(+) java.util.Spliterator spliterator()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.Stream stream()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP[] toArray(org.apache.uima.jcas.cas.TOP[])
	---  REMOVED METHOD: PUBLIC(-) java.lang.String[] toStringArray()
+++! NEW CLASS: PUBLIC(+) org.apache.uima.jcas.cas.FSArrayList  (not serializable)
	+++  NEW INTERFACE: java.util.RandomAccess
	+++  NEW INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	+++  NEW INTERFACE: org.apache.uima.UimaSerializable
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: java.util.List
	+++  NEW INTERFACE: org.apache.uima.UimaSerializableFSs
	+++  NEW INTERFACE: org.apache.uima.cas.CommonArrayFS
	+++  NEW INTERFACE: java.util.Collection
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.SelectViaCopyToArray
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: java.lang.Iterable
	+++  NEW SUPERCLASS: org.apache.uima.jcas.cas.TOP
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int typeIndexID
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int type
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_fsArray
	+++  NEW CONSTRUCTOR: PROTECTED(+) FSArrayList()
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSArrayList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSArrayList(org.apache.uima.jcas.JCas, int)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSArrayList(org.apache.uima.jcas.JCas)
	+++  NEW METHOD: PUBLIC(+) void _init_from_cas_data()
	+++  NEW METHOD: PUBLIC(+) void _save_to_cas_data()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.impl.FeatureStructureImplC _superClone()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.FeatureStructure[] _toArrayForSelect()
	+++  NEW METHOD: PUBLIC(+) boolean add(org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) void add(int, org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) boolean addAll(java.util.Collection)
	+++  NEW METHOD: PUBLIC(+) boolean addAll(int, java.util.Collection)
	+++  NEW METHOD: PUBLIC(+) void clear()
	+++  NEW METHOD: PUBLIC(+) boolean contains(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) boolean containsAll(java.util.Collection)
	+++  NEW METHOD: PUBLIC(+) void copyFromArray(org.apache.uima.cas.FeatureStructure[], int, int, int)
	+++  NEW METHOD: PUBLIC(+) void copyFromArray(java.lang.String[], int, int, int)
	+++  NEW METHOD: PUBLIC(+) void copyToArray(int, org.apache.uima.cas.FeatureStructure[], int, int)
	+++  NEW METHOD: PUBLIC(+) void copyToArray(int, java.lang.String[], int, int)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.FSArrayList create(org.apache.uima.jcas.JCas, org.apache.uima.cas.FeatureStructure[])
	+++  NEW METHOD: PUBLIC(+) boolean equals(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) void forEach(java.util.function.Consumer)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP get(int)
	+++  NEW METHOD: PUBLIC(+) int getTypeIndexID()
	+++  NEW METHOD: PUBLIC(+) int hashCode()
	+++  NEW METHOD: PUBLIC(+) int indexOf(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) boolean isEmpty()
	+++  NEW METHOD: PUBLIC(+) java.util.Iterator iterator()
	+++  NEW METHOD: PUBLIC(+) int lastIndexOf(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) java.util.ListIterator listIterator(int)
	+++  NEW METHOD: PUBLIC(+) java.util.ListIterator listIterator()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.Stream parallelStream()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP remove(int)
	+++  NEW METHOD: PUBLIC(+) boolean remove(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) boolean removeAll(java.util.Collection)
	+++  NEW METHOD: PUBLIC(+) boolean removeIf(java.util.function.Predicate)
	+++  NEW METHOD: PUBLIC(+) void replaceAll(java.util.function.UnaryOperator)
	+++  NEW METHOD: PUBLIC(+) boolean retainAll(java.util.Collection)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP set(int, org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) int size()
	+++  NEW METHOD: PUBLIC(+) void sort(java.util.Comparator)
	+++  NEW METHOD: PUBLIC(+) java.util.Spliterator spliterator()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.Stream stream()
	+++  NEW METHOD: PUBLIC(+) java.util.List subList(int, int)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP[] toArray()
	+++  NEW METHOD: PUBLIC(+) java.lang.Object[] toArray(java.lang.Object[])
	+++  NEW METHOD: PUBLIC(+) java.lang.String toString()
+++! NEW CLASS: PUBLIC(+) org.apache.uima.jcas.cas.FSHashSet  (not serializable)
	+++  NEW INTERFACE: java.util.RandomAccess
	+++  NEW INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	+++  NEW INTERFACE: org.apache.uima.UimaSerializable
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: org.apache.uima.UimaSerializableFSs
	+++  NEW INTERFACE: java.util.Collection
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.SelectViaCopyToArray
	+++  NEW INTERFACE: java.util.Set
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: java.lang.Iterable
	+++  NEW SUPERCLASS: org.apache.uima.jcas.cas.TOP
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int typeIndexID
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int type
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_fsArray
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSHashSet(org.apache.uima.jcas.JCas, int)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSHashSet(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW CONSTRUCTOR: PROTECTED(+) FSHashSet()
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSHashSet(java.util.HashSet, org.apache.uima.jcas.JCas, int)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSHashSet(org.apache.uima.jcas.JCas)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSHashSet(java.util.HashSet, org.apache.uima.jcas.JCas)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSHashSet(java.util.HashSet, org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) void _init_from_cas_data()
	+++  NEW METHOD: PUBLIC(+) void _save_to_cas_data()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.impl.FeatureStructureImplC _superClone()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP[] _toArrayForSelect()
	+++  NEW METHOD: PUBLIC(+) boolean add(org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) boolean addAll(java.util.Collection)
	+++  NEW METHOD: PUBLIC(+) void clear()
	+++  NEW METHOD: PUBLIC(+) boolean contains(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) boolean containsAll(java.util.Collection)
	+++  NEW METHOD: PUBLIC(+) boolean equals(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) int getTypeIndexID()
	+++  NEW METHOD: PUBLIC(+) int hashCode()
	+++  NEW METHOD: PUBLIC(+) boolean isEmpty()
	+++  NEW METHOD: PUBLIC(+) java.util.Iterator iterator()
	+++  NEW METHOD: PUBLIC(+) boolean remove(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) boolean removeAll(java.util.Collection)
	+++  NEW METHOD: PUBLIC(+) boolean retainAll(java.util.Collection)
	+++  NEW METHOD: PUBLIC(+) int size()
	+++  NEW METHOD: PUBLIC(+) java.util.Spliterator spliterator()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP[] toArray()
	+++  NEW METHOD: PUBLIC(+) java.lang.Object[] toArray(java.lang.Object[])
	+++  NEW METHOD: PUBLIC(+) java.lang.String toString()
+++! NEW CLASS: PUBLIC(+) org.apache.uima.jcas.cas.FSLinkedHashSet  (not serializable)
	+++  NEW INTERFACE: java.util.RandomAccess
	+++  NEW INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	+++  NEW INTERFACE: org.apache.uima.UimaSerializable
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: org.apache.uima.UimaSerializableFSs
	+++  NEW INTERFACE: java.util.Collection
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.SelectViaCopyToArray
	+++  NEW INTERFACE: java.util.Set
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: java.lang.Iterable
	+++  NEW SUPERCLASS: org.apache.uima.jcas.cas.FSHashSet
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int typeIndexID
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int type
	+++  NEW CONSTRUCTOR: PROTECTED(+) FSLinkedHashSet()
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSLinkedHashSet(org.apache.uima.jcas.JCas)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSLinkedHashSet(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSLinkedHashSet(org.apache.uima.jcas.JCas, int)
	+++  NEW METHOD: PUBLIC(+) int getTypeIndexID()
***! MODIFIED CLASS: PUBLIC ABSTRACT (<- NON_ABSTRACT) org.apache.uima.jcas.cas.FSList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonList
	---  REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) int typeIndexID
	---  REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) int type
	---! REMOVED CONSTRUCTOR: PUBLIC(-) FSList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) boolean contains(org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.FSList create(org.apache.uima.jcas.JCas, org.apache.uima.cas.FeatureStructure[])
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.NonEmptyFSList createNonEmptyNode()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyFSList emptyList()
	---  REMOVED METHOD: PUBLIC(-) int getTypeIndexID()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.NonEmptyFSList pushNode()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(org.apache.uima.cas.Type)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(java.lang.Class)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(int)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) java.util.stream.Stream stream()
+++! NEW CLASS: PUBLIC(+) org.apache.uima.jcas.cas.Int2FS  (not serializable)
	+++  NEW INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	+++  NEW INTERFACE: org.apache.uima.UimaSerializable
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: org.apache.uima.UimaSerializableFSs
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW SUPERCLASS: org.apache.uima.jcas.cas.TOP
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int typeIndexID
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int type
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_fsArray
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_intArray
	+++  NEW CONSTRUCTOR: PUBLIC(+) Int2FS(org.apache.uima.jcas.JCas, int)
	+++  NEW CONSTRUCTOR: PROTECTED(+) Int2FS()
	+++  NEW CONSTRUCTOR: PUBLIC(+) Int2FS(org.apache.uima.jcas.JCas)
	+++  NEW CONSTRUCTOR: PUBLIC(+) Int2FS(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) void _init_from_cas_data()
	+++  NEW METHOD: PUBLIC(+) void _save_to_cas_data()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.impl.FeatureStructureImplC _superClone()
	+++  NEW METHOD: PUBLIC(+) void clear()
	+++  NEW METHOD: PUBLIC(+) boolean containsKey(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) boolean containsValue(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP get(int)
	+++  NEW METHOD: PUBLIC(+) int getTypeIndexID()
	+++  NEW METHOD: PUBLIC(+) boolean isEmpty()
	+++  NEW METHOD: PUBLIC(+) java.util.Iterator iterator()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP put(int, org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP remove(int)
	+++  NEW METHOD: PUBLIC(+) int size()
	+++  NEW METHOD: PUBLIC(+) java.lang.String toString()
	+++  NEW METHOD: PUBLIC(+) java.util.Collection values()
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.IntegerArray  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonPrimitiveArray
	+++  NEW INTERFACE: org.apache.uima.cas.impl.IntArrayFSImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) IntegerArray(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) IntegerArray(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int)
	+++  NEW METHOD: PUBLIC(+) int[] _getTheArray()
	+++  NEW METHOD: PUBLIC(+) boolean contains(int)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.IntegerArray create(org.apache.uima.jcas.JCas, int[])
	+++  NEW METHOD: PUBLIC(+) void forEach(java.util.function.IntConsumer)
	+++  NEW METHOD: PUBLIC(+) void setArrayValueFromString(int, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) java.util.Spliterator$OfInt spliterator()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.IntStream stream()
	---  REMOVED METHOD: PUBLIC(-) java.lang.String[] toStringArray()
+++! NEW CLASS: PUBLIC(+) org.apache.uima.jcas.cas.IntegerArrayList  (not serializable)
	+++  NEW INTERFACE: java.util.RandomAccess
	+++  NEW INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	+++  NEW INTERFACE: org.apache.uima.UimaSerializable
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: org.apache.uima.cas.CommonArrayFS
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: java.lang.Iterable
	+++  NEW SUPERCLASS: org.apache.uima.jcas.cas.TOP
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int typeIndexID
	+++! NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int type
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_intArray
	+++  NEW CONSTRUCTOR: PUBLIC(+) IntegerArrayList(org.apache.uima.jcas.JCas, int)
	+++  NEW CONSTRUCTOR: PUBLIC(+) IntegerArrayList(org.apache.uima.jcas.JCas)
	+++  NEW CONSTRUCTOR: PROTECTED(+) IntegerArrayList()
	+++  NEW CONSTRUCTOR: PUBLIC(+) IntegerArrayList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) void _init_from_cas_data()
	+++  NEW METHOD: PUBLIC(+) void _save_to_cas_data()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.impl.FeatureStructureImplC _superClone()
	+++  NEW METHOD: PUBLIC(+) boolean add(int)
	+++  NEW METHOD: PUBLIC(+) void add(int, int)
	+++  NEW METHOD: PUBLIC(+) void clear()
	+++  NEW METHOD: PUBLIC(+) boolean contains(int)
	+++  NEW METHOD: PUBLIC(+) void copyFromArray(int[], int, int, int)
	+++  NEW METHOD: PUBLIC(+) void copyFromArray(java.lang.String[], int, int, int)
	+++  NEW METHOD: PUBLIC(+) void copyToArray(int, int[], int, int)
	+++  NEW METHOD: PUBLIC(+) void copyToArray(int, java.lang.String[], int, int)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.IntegerArrayList create(org.apache.uima.jcas.JCas, int[])
	+++  NEW METHOD: PUBLIC(+) boolean equals(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) void forEach(java.util.function.IntConsumer)
	+++  NEW METHOD: PUBLIC(+) int get(int)
	+++  NEW METHOD: PUBLIC(+) int getTypeIndexID()
	+++  NEW METHOD: PUBLIC(+) int hashCode()
	+++  NEW METHOD: PUBLIC(+) int indexOf(int)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.internal.util.IntListIterator intListIterator()
	+++  NEW METHOD: PUBLIC(+) java.util.PrimitiveIterator$OfInt iterator()
	+++  NEW METHOD: PUBLIC(+) int lastIndexOf(int)
	+++  NEW METHOD: PUBLIC(+) boolean remove(int)
	+++  NEW METHOD: PUBLIC(+) int removeAt(int)
	+++  NEW METHOD: PUBLIC(+) void set(int, int)
	+++  NEW METHOD: PUBLIC(+) int size()
	+++  NEW METHOD: PUBLIC(+) void sort()
	+++  NEW METHOD: PUBLIC(+) java.util.Spliterator$OfInt spliterator()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.IntStream stream()
	+++  NEW METHOD: PUBLIC(+) int[] toArray()
	+++  NEW METHOD: PUBLIC(+) int[] toArray(int[])
	+++  NEW METHOD: PUBLIC(+) java.lang.String toString()
***! MODIFIED CLASS: PUBLIC ABSTRACT (<- NON_ABSTRACT) org.apache.uima.jcas.cas.IntegerList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonList
	---  REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) int typeIndexID
	---  REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) int type
	+++  NEW FIELD: PUBLIC(+) STATIC(+) java.util.PrimitiveIterator$OfInt EMPTY_INT_ITERATOR
	---! REMOVED CONSTRUCTOR: PUBLIC(-) IntegerList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) IntegerList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) boolean contains(int)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.IntegerList create(org.apache.uima.jcas.JCas, int[])
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.NonEmptyIntegerList createNonEmptyNode()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyIntegerList emptyList()
	---  REMOVED METHOD: PUBLIC(-) int getTypeIndexID()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.NonEmptyIntegerList pushNode()
	+++  NEW METHOD: PUBLIC(+) java.util.Spliterator$OfInt spliterator()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.Stream stream()
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.LongArray  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonPrimitiveArray
	+++  NEW INTERFACE: org.apache.uima.cas.impl.LongArrayFSImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) LongArray(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) LongArray(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int)
	+++  NEW METHOD: PUBLIC(+) long[] _getTheArray()
	+++  NEW METHOD: PUBLIC(+) boolean contains(long)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.LongArray create(org.apache.uima.jcas.JCas, long[])
	+++  NEW METHOD: PUBLIC(+) void forEach(java.util.function.LongConsumer)
	+++  NEW METHOD: PUBLIC(+) void setArrayValueFromString(int, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) java.util.Spliterator$OfLong spliterator()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.LongStream stream()
	---  REMOVED METHOD: PUBLIC(-) java.lang.String[] toStringArray()
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.NonEmptyFloatList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.NonEmptyList
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_tail
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_head
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyFloatList(org.apache.uima.jcas.JCas, float, org.apache.uima.jcas.cas.FloatList)
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyFloatList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyFloatList(org.apache.uima.jcas.JCas, float)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyFloatList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyFloatList(org.apache.uima.jcas.JCas, float, org.apache.uima.jcas.cas.CommonList)
	+++  NEW METHOD: PUBLIC(+) java.lang.String get_headAsString()
	+++  NEW METHOD: PUBLIC(+) void set_headFromString(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void setTail(org.apache.uima.jcas.cas.CommonList)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.NonEmptyFSList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.NonEmptyList
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_tail
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_head
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyFSList(int, org.apache.uima.jcas.cas.TOP_Type)
	***  MODIFIED CONSTRUCTOR: PUBLIC (<- PROTECTED) NonEmptyFSList()
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyFSList(org.apache.uima.jcas.JCas, org.apache.uima.jcas.cas.TOP, org.apache.uima.jcas.cas.FSList)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyFSList(org.apache.uima.jcas.JCas, org.apache.uima.jcas.cas.TOP)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyFSList(org.apache.uima.jcas.JCas, org.apache.uima.jcas.cas.TOP, org.apache.uima.jcas.cas.CommonList)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyFSList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.NonEmptyFSList add(org.apache.uima.jcas.cas.TOP)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP getNthElement(int)
	+++  NEW METHOD: PUBLIC(+) void setTail(org.apache.uima.jcas.cas.CommonList)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.NonEmptyIntegerList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.NonEmptyList
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_tail
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_head
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyIntegerList(org.apache.uima.jcas.JCas, int, org.apache.uima.jcas.cas.IntegerList)
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyIntegerList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyIntegerList(org.apache.uima.jcas.JCas, int)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyIntegerList(org.apache.uima.jcas.JCas, int, org.apache.uima.jcas.cas.CommonList)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyIntegerList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) java.lang.String get_headAsString()
	+++  NEW METHOD: PUBLIC(+) void set_headFromString(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void setHead(java.util.List, int)
	+++  NEW METHOD: PUBLIC(+) void setTail(org.apache.uima.jcas.cas.CommonList)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.jcas.cas.NonEmptyList  (not serializable)
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonList
	+++  NEW INTERFACE: java.lang.Cloneable
	+++  NEW INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW SUPERCLASS: java.lang.Object
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.NonEmptyStringList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.NonEmptyList
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_tail
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_head
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyStringList(org.apache.uima.jcas.JCas, java.lang.String, org.apache.uima.jcas.cas.StringList)
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyStringList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyStringList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyStringList(org.apache.uima.jcas.JCas, java.lang.String)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyStringList(org.apache.uima.jcas.JCas, java.lang.String, org.apache.uima.jcas.cas.CommonList)
	+++  NEW METHOD: PUBLIC(+) java.lang.String get_headAsString()
	+++  NEW METHOD: PUBLIC(+) void set_headFromString(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void setTail(org.apache.uima.jcas.cas.CommonList)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.jcas.cas.SelectViaCopyToArray  (not serializable)
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.impl.CASImpl _getView()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.FeatureStructure[] _toArrayForSelect()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.impl.SelectFSs_impl select()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(org.apache.uima.cas.Type)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(java.lang.Class)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(int)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(java.lang.String)
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.ShortArray  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonPrimitiveArray
	+++  NEW INTERFACE: org.apache.uima.cas.impl.ShortArrayFSImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) ShortArray(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) ShortArray(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int)
	+++  NEW METHOD: PUBLIC(+) short[] _getTheArray()
	+++  NEW METHOD: PUBLIC(+) boolean contains(short)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.ShortArray create(org.apache.uima.jcas.JCas, short[])
	+++  NEW METHOD: PUBLIC(+) void setArrayValueFromString(int, java.lang.String)
	---  REMOVED METHOD: PUBLIC(-) java.lang.String[] toStringArray()
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.Sofa  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.SofaFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.cas.impl.SofaFSImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_sofaArray
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_sofaString
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_sofaNum
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_sofaID
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_sofaURI
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_mimeType
	---! REMOVED CONSTRUCTOR: PUBLIC(-) Sofa(int, org.apache.uima.jcas.cas.TOP_Type)
	---! REMOVED CONSTRUCTOR: PUBLIC(-) Sofa(org.apache.uima.jcas.JCas, org.apache.uima.cas.SofaID, java.lang.String)
		---  REMOVED ANNOTATION: java.lang.Deprecated
	+++  NEW CONSTRUCTOR: PUBLIC(+) Sofa(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int, java.lang.String, java.lang.String)
	+++  NEW CONSTRUCTOR: PUBLIC(+) Sofa(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.TOP getSofaArray()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getSofaString()
	***  MODIFIED METHOD: PUBLIC (<- PRIVATE) boolean isSofaDataSet()
	+++  NEW METHOD: PUBLIC(+) void setLocalSofaData(org.apache.uima.cas.FeatureStructure, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void setLocalSofaData(java.lang.String, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void setLocalSofaDataNoDocAnnotUpdate(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void setMimeType(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void setRemoteSofaURI(java.lang.String, java.lang.String)
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.StringArray  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.CommonArrayFS
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonPrimitiveArray
	+++  NEW INTERFACE: org.apache.uima.cas.impl.StringArrayFSImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) StringArray(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) StringArray(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl, int)
	+++  NEW METHOD: PUBLIC(+) java.lang.String[] _getTheArray()
	+++  NEW METHOD: PUBLIC(+) boolean contains(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void copyValuesFrom(org.apache.uima.cas.CommonArrayFS)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.StringArray create(org.apache.uima.jcas.JCas, java.lang.String[])
	+++  NEW METHOD: PUBLIC(+) void setArrayValueFromString(int, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) java.util.stream.Stream stream()
	---  REMOVED METHOD: PUBLIC(-) java.lang.String[] toStringArray()
***! MODIFIED CLASS: PUBLIC ABSTRACT (<- NON_ABSTRACT) org.apache.uima.jcas.cas.StringList  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.jcas.cas.CommonList
	---  REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) int typeIndexID
	---  REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) int type
	---! REMOVED CONSTRUCTOR: PUBLIC(-) StringList(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) StringList(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) boolean contains(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.StringList create(org.apache.uima.jcas.JCas, java.lang.String[])
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.NonEmptyStringList createNonEmptyNode()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyStringList emptyList()
	---  REMOVED METHOD: PUBLIC(-) int getTypeIndexID()
	+++  NEW METHOD: PUBLIC(+) java.util.stream.Stream stream()
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.TOP  (not serializable)
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	***! MODIFIED SUPERCLASS: org.apache.uima.cas.impl.FeatureStructureImplC (<- org.apache.uima.cas.impl.FeatureStructureImpl)
	---  REMOVED FIELD: PROTECTED(-) FINAL(-) int addr
	---! REMOVED FIELD: PUBLIC(-) org.apache.uima.jcas.cas.TOP_Type jcasType
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.jcas.cas.TOP _singleton
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	---! REMOVED CONSTRUCTOR: PUBLIC(-) TOP(int, org.apache.uima.jcas.cas.TOP_Type)
	***  MODIFIED CONSTRUCTOR: PUBLIC (<- PROTECTED) TOP()
	+++  NEW CONSTRUCTOR: PUBLIC(+) TOP(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.TOP _createJCasHashMapReserve(int)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.jcas.cas.TOP _createSearchKey(int)
	---  REMOVED METHOD: PUBLIC(-) void addToIndexes()
	---  REMOVED METHOD: PUBLIC(-) void addToIndexes(org.apache.uima.jcas.JCas)
	---  REMOVED METHOD: PUBLIC(-) boolean equals(java.lang.Object)
	---  REMOVED METHOD: PUBLIC(-) int getAddress()
	---  REMOVED METHOD: PUBLIC(-) org.apache.uima.cas.CAS getCAS()
	---  REMOVED METHOD: PUBLIC(-) org.apache.uima.cas.impl.CASImpl getCASImpl()
	---  REMOVED METHOD: PUBLIC(-) org.apache.uima.cas.impl.LowLevelCAS getLowLevelCas()
	---  REMOVED METHOD: PUBLIC(-) int hashCode()
	---  REMOVED METHOD: PUBLIC(-) void removeFromIndexes()
	---  REMOVED METHOD: PUBLIC(-) void removeFromIndexes(org.apache.uima.jcas.JCas)
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.jcas.JCas  (not serializable)
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) void checkArrayBounds(int, int)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.CommonArrayFS emptyArray(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.BooleanArray emptyBooleanArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.ByteArray emptyByteArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.DoubleArray emptyDoubleArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.FloatArray emptyFloatArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyFloatList emptyFloatList()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.FSArray emptyFSArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyFSList emptyFSList()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.IntegerArray emptyIntegerArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyIntegerList emptyIntegerList()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyList emptyList(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.LongArray emptyLongArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.ShortArray emptyShortArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.StringArray emptyStringArray()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.EmptyStringList emptyStringList()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.FSIterator getAllIndexedFS(org.apache.uima.cas.Type)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.Type getCasType(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.FloatArray getFloatArray0L()
		+++  NEW ANNOTATION: java.lang.Deprecated
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) org.apache.uima.jcas.cas.FSArray getFSArray0L()
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++* NEW METHOD: PUBLIC(+) java.util.Collection getIndexedFSs(org.apache.uima.cas.Type)
	+++* NEW METHOD: PUBLIC(+) java.util.Collection getIndexedFSs(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) java.util.Collection getIndexedFSs()
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) org.apache.uima.jcas.cas.IntegerArray getIntegerArray0L()
		+++  NEW ANNOTATION: java.lang.Deprecated
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) org.apache.uima.jcas.cas.TOP getJfsFromCaddr(int)
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) org.apache.uima.cas.Feature getRequiredFeatureDE(org.apache.uima.cas.Type, java.lang.String, java.lang.String, boolean)
		+++  NEW ANNOTATION: java.lang.Deprecated
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) org.apache.uima.jcas.cas.StringArray getStringArray0L()
		+++  NEW ANNOTATION: java.lang.Deprecated
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) void putJfsFromCaddr(int, org.apache.uima.cas.FeatureStructure)
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++* NEW METHOD: PUBLIC(+) void removeAllExcludingSubtypes(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) void removeAllIncludingSubtypes(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select()
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(org.apache.uima.cas.Type)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(java.lang.Class)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(int)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.cas.SelectFSs select(java.lang.String)
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) void throwFeatMissing(java.lang.String, java.lang.String)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.JCasRegistry  (not serializable)
	***! MODIFIED CONSTRUCTOR: PRIVATE (<- PUBLIC) JCasRegistry()
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.tcas.Annotation  (not serializable)
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.FeatureStructureImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.text.AnnotationFS
	===! UNCHANGED INTERFACE: org.apache.uima.cas.impl.AnnotationBaseImpl
	===* UNCHANGED INTERFACE: org.apache.uima.cas.FeatureStructure
	+++  NEW INTERFACE: org.apache.uima.cas.impl.AnnotationImpl
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _TypeName
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_begin
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String _FeatName_end
	===  UNCHANGED CONSTRUCTOR: PROTECTED Annotation()
		+++  NEW ANNOTATION: java.lang.Deprecated
	---! REMOVED CONSTRUCTOR: PUBLIC(-) Annotation(int, org.apache.uima.jcas.cas.TOP_Type)
	+++  NEW CONSTRUCTOR: PUBLIC(+) Annotation(org.apache.uima.cas.impl.TypeImpl, org.apache.uima.cas.impl.CASImpl)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) int compareAnnotation(org.apache.uima.jcas.tcas.Annotation)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) int compareAnnotation(org.apache.uima.jcas.tcas.Annotation, org.apache.uima.cas.admin.LinearTypeOrder)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) int compareAnnotationWithId(org.apache.uima.jcas.tcas.Annotation)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) int compareAnnotationWithId(org.apache.uima.jcas.tcas.Annotation, org.apache.uima.cas.admin.LinearTypeOrder)
	***! MODIFIED METHOD: PUBLIC FINAL (<- NON_FINAL) int getBegin()
	***! MODIFIED METHOD: PUBLIC FINAL (<- NON_FINAL) int getEnd()
	***! MODIFIED METHOD: PUBLIC FINAL (<- NON_FINAL) void setBegin(int)
	***! MODIFIED METHOD: PUBLIC FINAL (<- NON_FINAL) void setEnd(int)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.List_of_ints  (not serializable)
	+++  NEW INTERFACE: java.lang.Iterable
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) boolean add(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void add(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void clear()
	+++  NEW METHOD: PUBLIC(+) boolean contains(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void copyFromArray(int[], int, int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void copyToArray(int, int[], int, int)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.List_of_ints EMPTY_LIST()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) int get(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) int indexOf(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.internal.util.IntListIterator intListIterator()
	+++  NEW METHOD: PUBLIC(+) boolean isEmpty()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.util.PrimitiveIterator$OfInt iterator()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) int lastIndexOf(int)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.List_of_ints newInstance(int[])
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.List_of_ints newInstance(int[], int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) boolean remove(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) int removeAtIndex(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) int set(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) int size()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void sort()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.List_of_ints subList(int, int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) int[] toArray()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) int[] toArrayMinCopy()
***  MODIFIED CLASS: PUBLIC STATIC org.apache.uima.pear.tools.InstallationDescriptor$ComponentInfo  (field changed from nontransient to transient)
	***  MODIFIED FIELD: PUBLIC org.apache.uima.pear.tools.InstallationDescriptor$ServiceInfo serviceInfo
***  MODIFIED CLASS: PUBLIC org.apache.uima.resource.CasDefinition  (not serializable)
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.impl.TypeSystemImpl getTypeSystemImpl()
	+++  NEW METHOD: PUBLIC(+) void setTypeSystemImpl(org.apache.uima.cas.impl.TypeSystemImpl)
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.uima.resource.Resource_ImplBase  (not serializable)
	+++  NEW METHOD: PUBLIC(+) java.lang.Class loadUserClass(java.lang.String)
		+++  NEW EXCEPTION: java.lang.ClassNotFoundException
	+++  NEW METHOD: PUBLIC(+) java.lang.Class loadUserClassOrThrow(java.lang.String, org.apache.uima.resource.ResourceSpecifier)
		+++  NEW EXCEPTION: org.apache.uima.resource.ResourceInitializationException
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.UimaContext setContextHolder()
	+++  NEW METHOD: PUBLIC(+) void setContextHolderX(org.apache.uima.internal.util.function.Runnable_withException)
		+++  NEW EXCEPTION: java.lang.Exception
	+++  NEW METHOD: PUBLIC(+) void withContextHolder(java.lang.Runnable)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.resource.ResourceAccessException  (compatible)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.resource.ResourceConfigurationException  (compatible)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.resource.ResourceInitializationException  (compatible)
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.ResourceManager  (not serializable)
	+++* NEW METHOD: PUBLIC(+) void setExtensionClassLoader(java.lang.ClassLoader, boolean)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.resource.ResourceProcessException  (compatible)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.resource.ResourceServiceException  (compatible)
===* UNCHANGED CLASS: PUBLIC org.apache.uima.search.IndexingException  (compatible)
===! UNCHANGED CLASS: PUBLIC org.apache.uima.UIMA_IllegalArgumentException  (compatible)
***! MODIFIED CLASS: PUBLIC org.apache.uima.UIMA_IllegalStateException  (compatible)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_NO_TYPE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JCAS_INCOMPATIBLE_TYPE_SYSTEMS
===! UNCHANGED CLASS: PUBLIC org.apache.uima.UIMA_UnsupportedOperationException  (compatible)
***! MODIFIED CLASS: PUBLIC org.apache.uima.UIMAException  (superclass modified)
	+++  NEW INTERFACE: org.apache.uima.I18nExceptionI
	***! MODIFIED SUPERCLASS: java.lang.Exception (<- org.apache.uima.InternationalizedException)
	+++  NEW METHOD: PUBLIC(+) java.lang.Object[] getArguments()
	+++  NEW METHOD: PUBLIC(+) java.lang.Throwable getCause()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getLocalizedMessage()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getMessage()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getMessageKey()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getResourceBundleName()
	+++  NEW METHOD: PUBLIC(+) java.lang.Throwable initCause(java.lang.Throwable)
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.uima.UIMAFramework  (not serializable)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SKIP_USER_JCAS_LOADING
***! MODIFIED CLASS: PUBLIC org.apache.uima.UIMARuntimeException  (superclass modified)
	+++  NEW INTERFACE: org.apache.uima.I18nExceptionI
	***! MODIFIED SUPERCLASS: java.lang.RuntimeException (<- org.apache.uima.InternationalizedRuntimeException)
	---! REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) java.lang.String STANDARD_MESSAGE_CATALOG
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String resource_file
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String NOT_SUPPORTED_NO_HEAP_IN_UIMA_V3
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String INTERNAL_ERROR
	---! REMOVED CONSTRUCTOR: PUBLIC(-) UIMARuntimeException(java.lang.String, java.lang.String, java.lang.Object[], java.lang.Throwable)
	---! REMOVED CONSTRUCTOR: PUBLIC(-) UIMARuntimeException(java.lang.String, java.lang.Object[], java.lang.Throwable)
	+++  NEW CONSTRUCTOR: PUBLIC(+) UIMARuntimeException(java.lang.Throwable, java.lang.String, java.lang.Object[])
	+++  NEW CONSTRUCTOR: PUBLIC(+) UIMARuntimeException(java.lang.Throwable, java.lang.String, java.lang.String, java.lang.Object[])
	+++  NEW METHOD: PUBLIC(+) java.lang.Object[] getArguments()
	+++  NEW METHOD: PUBLIC(+) java.lang.Throwable getCause()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getLocalizedMessage()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getMessage()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getMessageKey()
	+++  NEW METHOD: PUBLIC(+) java.lang.String getResourceBundleName()
	+++  NEW METHOD: PUBLIC(+) java.lang.Throwable initCause(java.lang.Throwable)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.UimaSerializable  (not serializable)
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void _init_from_cas_data()
	+++  NEW METHOD: PUBLIC(+) void _reset_cas_data()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void _save_to_cas_data()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.uima.cas.impl.FeatureStructureImplC _superClone()
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.cas.impl.FeatureStructureImplC clone()
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.UimaSerializableFSs  (not serializable)
	+++  NEW INTERFACE: org.apache.uima.UimaSerializable
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) void _save_fsRefs_to_cas_data()
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.util.AutoCloseableNoException  (not serializable)
	+++  NEW INTERFACE: java.lang.AutoCloseable
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void close()
***  MODIFIED CLASS: PUBLIC org.apache.uima.util.CasCopier  (not serializable)
	+++  NEW METHOD: PUBLIC(+) boolean alreadyCopied(org.apache.uima.jcas.cas.TOP)
***  MODIFIED CLASS: PUBLIC org.apache.uima.util.FileUtils  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) void writeToFile(java.nio.file.Path, java.lang.String)
+++  NEW CLASS: PUBLIC(+) org.apache.uima.util.IntEntry  (not serializable)
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW CONSTRUCTOR: PUBLIC(+) IntEntry(int, java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) boolean equals(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) int getKey()
	+++  NEW METHOD: PUBLIC(+) java.lang.Object getValue()
	+++  NEW METHOD: PUBLIC(+) int hashCode()
===* UNCHANGED CLASS: PUBLIC org.apache.uima.util.InvalidXMLException  (compatible)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.util.IteratorNvc  (not serializable)
	+++  NEW INTERFACE: java.util.Iterator
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.Object nextNvc()
***  MODIFIED CLASS: PUBLIC org.apache.uima.util.Level  (not serializable)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int DEBUG_INT
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.util.Level ERROR
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.util.Level TRACE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int TRACE_INT
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.util.Level DEBUG
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.uima.util.Level WARN
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int WARN_INT
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int ERROR_INT
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.util.Logger  (not serializable)
	+++  NEW INTERFACE: org.slf4j.Logger
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.slf4j.Marker UIMA_MARKER_CONFIG
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.slf4j.Marker UIMA_MARKER_FINEST
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void debug(java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void debug(java.util.function.Supplier, java.lang.Throwable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void debug(org.slf4j.Marker, java.lang.String, java.util.function.Supplier[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void debug(java.lang.String, java.util.function.Supplier[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void debug(org.slf4j.Marker, java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void debug(org.slf4j.Marker, java.util.function.Supplier, java.lang.Throwable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void error(java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void error(java.util.function.Supplier, java.lang.Throwable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void error(org.slf4j.Marker, java.lang.String, java.util.function.Supplier[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void error(java.lang.String, java.util.function.Supplier[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void error(org.slf4j.Marker, java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void error(org.slf4j.Marker, java.util.function.Supplier, java.lang.Throwable)
	+++* NEW METHOD: PUBLIC(+) org.apache.uima.util.Logger getLimitedLogger(int)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void info(java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void info(java.util.function.Supplier, java.lang.Throwable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void info(org.slf4j.Marker, java.lang.String, java.util.function.Supplier[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void info(org.slf4j.Marker, java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void info(org.slf4j.Marker, java.util.function.Supplier, java.lang.Throwable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) boolean isAnnotatorLogger()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) boolean isLoggable(org.apache.uima.util.Level, org.slf4j.Marker)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String rb(java.lang.String, java.lang.String, java.lang.Object[])
	+++* NEW METHOD: PUBLIC(+) java.lang.String rb_ue(java.lang.String, java.lang.Object[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void trace(java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void trace(java.util.function.Supplier, java.lang.Throwable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void trace(org.slf4j.Marker, java.lang.String, java.util.function.Supplier[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void trace(java.lang.String, java.util.function.Supplier[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void trace(org.slf4j.Marker, java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void trace(org.slf4j.Marker, java.util.function.Supplier, java.lang.Throwable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void warn(java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void warn(java.util.function.Supplier, java.lang.Throwable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void warn(org.slf4j.Marker, java.lang.String, java.util.function.Supplier[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void warn(java.lang.String, java.util.function.Supplier[])
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void warn(org.slf4j.Marker, java.util.function.Supplier)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void warn(org.slf4j.Marker, java.util.function.Supplier, java.lang.Throwable)
